### PR TITLE
Fixes #56: WHOIS RIPE analyzer

### LINKED ISSUE #56: RIPE API analyzer
Get WHOIS for a specific IP with the RIPE API:

https://github.com/RIPE-NCC/whois/wiki/WHOIS-REST-API
https://www.ripe.net/manage-ips-and-asns/db/support/documentation/ripe-database-query-reference-manual#tables-of-query-types-supported-by-the-ripe-database
http://jaluther.blogspot.com/2016/04/python-for-network-engineers-part-2.html

### PR DESCRIPTION
**Pull request by 0x0elliot**
# Description

RIPE API WHOIS lookup analyzer added.
## Related issues

<a href = "https://github.com/intelowlproject/IntelOwl/issues/56">Here</a> is the issue.

## Type of change

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality).

# Checklist

- [x] The pull request is for the branch develop
- [x] A new analyzer or connector was added, in which case:
    - [x] [Usage](https://github.com/intelowlproject/IntelOwl/blob/master/docs/source/Usage.md) file was updated.
    - [x] If the analyzer/connector requires mocked testing, `_monkeypatch()` was used in it's class to apply the necessary decorators.

## Screenshot of output:
Screenshot 2021-11-15 at 1 10 39 PM

### CODE REVIEW

File: api_app/analyzers_manager/observable_analyzers/whoisripe.py [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+
+import requests
+
+
+from api_app.analyzers_manager import classes
+
+from tests.mock_utils import if_mock_connections, patch, MockResponse
+
+class Whoisripeapi(classes.ObservableAnalyzer):
+    url: str = "https://rest.db.ripe.net/search.json"
```
Comment (eshaan7) [inline api_app/analyzers_manager/observable_analyzers/whoisripe.py:12]: rename to `WhoIsRipeAPI`. change in python_module too.

File: api_app/analyzers_manager/observable_analyzers/whoisripe.py [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+# See the file 'LICENSE' for copying permission.
+
+import requests
+
+
+from api_app.analyzers_manager import classes
+
+from tests.mock_utils import if_mock_connections, patch, MockResponse
+
+class Whoisripeapi(classes.ObservableAnalyzer):
```
Comment (eshaan7) [inline api_app/analyzers_manager/observable_analyzers/whoisripe.py:11]: rename to `WhoIsRipeAPI`. change the same in python_module and docs too.

**Review by eshaan7 — CHANGES_REQUESTED** [review]
also un the [pre-commit](https://github.com/intelowlproject/IntelOwl/blob/master/.pre-commit-config.yaml) tools (black, flake8, isort) to lint the code.

File: api_app/analyzers_manager/observable_analyzers/whoisripe.py (hunk 2 above)
Comment (0x0elliot) [reply api_app/analyzers_manager/observable_analyzers/whoisripe.py:11]: Understood.

File: configuration/analyzer_config.json [hunk 3]
```diff
@@ -3275,6 +3275,23 @@
     },
     "params": {}
   },
+ "Whoisripe" : {
```
Comment (eshaan7) [inline configuration/analyzer_config.json:3278]: rename to `WhoIs_RipeDB_Search` and change in Usage.md docs.

File: configuration/analyzer_config.json [hunk 4]
```diff
@@ -3275,6 +3275,23 @@
...
+    "type": "observable",
+    "python_module" : "whoisripe.Whoisripeapi",
+    "description" : "The WHOIS record data of an IP address using RIPE API",
```
Comment (eshaan7) [inline configuration/analyzer_config.json:3281]: Change description to:

```
Fetch whois record data of an IP address from Ripe DB using their [search API](https://github.com/RIPE-NCC/whois/wiki/WHOIS-REST-API-search)
```

File: configuration/analyzer_config.json (hunk 3 above)
Comment (0x0elliot) [reply configuration/analyzer_config.json:3278]: Noted

File: configuration/analyzer_config.json (hunk 4 above)
Comment (0x0elliot) [reply configuration/analyzer_config.json:3281]: Noted.

**Comment by 0x0elliot** [comment]
Hopefully, I have added satisfying changes. Kindly review.

**Comment by 0x0elliot** [comment]
Let me do a quick rollback.

**Review by eshaan7 — APPROVED** [review]

**Pull request merged**